Packet++/header/IPv4Layer.h

```diff
@@ -298,7 +298,7 @@ namespace pcpp
 				if (res.type == IPv4TimestampOptionValue::TimestampAndIP)
 					readIPAddr = !readIPAddr;
 
-				valueOffset += (uint8_t)(4);
+				valueOffset += 4;
```

Comment:
I think removing the casting might cause a compilation warning on some platfroms